rest/taskana-rest-spring-example/src/main/java/pro/taskana/jobs/JobScheduler.java

```diff
@@ -0,0 +1,123 @@
+package pro.taskana.jobs;
+
+import java.security.Principal;
+import java.security.PrivilegedActionException;
+import java.security.PrivilegedExceptionAction;
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.ZoneId;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+import javax.security.auth.Subject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+import pro.taskana.BulkOperationResults;
+import pro.taskana.TaskanaEngine;
+import pro.taskana.TaskanaRole;
+import pro.taskana.impl.util.LoggerUtils;
+import pro.taskana.security.UserPrincipal;
+import pro.taskana.transaction.TaskanaTransactionProvider;
+
+/**
+ * This class invokes the JobRunner periodically to schedule long running jobs.
+ */
+@Component
+public class JobScheduler {
+
+    private final long untilDays = 14;
```

Comment:
Should untilDays be a configuration from taskana.properties?

Reply:
yes. but we can do this later ...